drivers/setup/blue/blue.c

```diff
@@ -1267,23 +1533,16 @@ ScrDispatch(
     _In_ PDEVICE_OBJECT DeviceObject,
     _In_ PIRP Irp)
 {
-    NTSTATUS Status;
     PIO_STACK_LOCATION stk = IoGetCurrentIrpStackLocation(Irp);
 
-    switch (stk->MajorFunction)
-    {
-        case IRP_MJ_CLOSE:
-            Status = STATUS_SUCCESS;
-            break;
+    UNREFERENCED_PARAMETER(DeviceObject);
 
-        default:
-            Status = STATUS_NOT_IMPLEMENTED;
-            break;
-    }
+    DPRINT1("ScrDispatch(0x%p): stk->MajorFunction = %lu UNIMPLEMENTED\n",
+            DeviceObject, stk->MajorFunction);
 
-    Irp->IoStatus.Status = Status;
+    Irp->IoStatus.Status = STATUS_NOT_IMPLEMENTED;
     IoCompleteRequest(Irp, IO_NO_INCREMENT);
-    return Status;
+    return STATUS_NOT_IMPLEMENTED;
```

Comment:
This function seems pretty useless now, unless you specifically need to return `STATUS_NOT_IMPLEMENTED`. If you don't assign a handler to one of the `MajorFunction` members, the default handler will complete the IRP with `STATUS_INVALID_DEVICE_REQUEST`.